Accessibility/FetchTimer/FetchTimer.csproj

```diff
@@ -2,6 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
   <PropertyGroup>
+    <TargetFrameworkVersion Condition="!$(MSBuildProjectName.Contains('netcore'))">v4.7.2</TargetFrameworkVersion>
```

Comment:
We don’t need the condition anymore

Reply:
Same note goes for other projects. The condition was useful in a props file. In the csproj, there is no need for it.

Reply:
I removed the condition from all csproj files.